src/c/pop-notification.js

```diff
@@ -3,12 +3,18 @@ import h from '../h';
 
 const popNotification = {
     controller(args) {
+        const displayNotification = args.toggleOpt || h.toggleProp(true, false),
+            setPopTimeout = () => {
+                setTimeout(() => { displayNotification(false); m.redraw(); }, 3000);
```

Comment:
We can't use toggle because if the user closes the pop-up before the timeout it will appear again.